### PR TITLE
bugfix: ignore prefix of Pretty.sharedOption in LLDB commands

### LINKED ISSUE #142: Bugfix: Ignore global label when use LLDB commands `_p` and `_pp`
Because redundant.

```swift
(lldb) _pp obj
🍎 [  // label is redundant
    1,
    2,
]
```

### PR DESCRIPTION
**Pull request by YusukeHosonuma**
resolved #142 

- [x] Update script 
- [x] Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -333,6 +333,9 @@ or install via [lowmad](https://github.com/bangerang/lowmad):
 $ lowmad install https://github.com/YusukeHosonuma/SwiftPrettyPrint.git
 ```
 
+**Note:**
+If you already install version 1.1.0 or older via [lowmad](https://github.com/bangerang/lowmad), please remove scripts manually before update. (e.g. `rm /usr/local/lib/lowmad/commands/YusukeHosonuma-SwiftPrettyPrint/swift_pretty_print.py`)
```
Comment (sahara-ooga) [inline README.md]: It says `version 1.1.0`, but I think it is hard to understand what version it is referring to.

File: README.md [hunk 2]
```diff
@@ -333,6 +333,9 @@ or install via [lowmad](https://github.com/bangerang/lowmad):
...
+
 This let you to use the lldb command in debug console as follows:
```
Comment (sahara-ooga) [inline README.md]:
```suggestion
This lets you to use the lldb command in debug console as follows:
```

I think it's like this :)

File: README.md (hunk 1 above)
Comment (YusukeHosonuma) [reply README.md]: Clarified.

```suggestion
If you already installed 1.1.0 or older version of SwiftPrettyPrint via [lowmad](https://github.com/bangerang/lowmad), please remove scripts manually before update. (e.g. `rm /usr/local/lib/lowmad/commands/YusukeHosonuma-SwiftPrettyPrint/swift_pretty_print.py`)
```

**Review by sahara-ooga — APPROVED** [review]
This looks good🙏🏻
Useful modification!

**Comment by YusukeHosonuma** [comment]
Thanks for reviews 👍

**Pull request merged**